<a href="https://colab.research.google.com/github/syuan611/115-1/blob/main/411403930_H6.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [3]:
a=float(input(" 請輸入身高cm:"))
b=float(input(" 請輸入體重kg:"))
bmi=b/(a*0.01)**2
if bmi<18.5:
  x="過輕"
elif bmi<24:
  x="正常"
else:
  x="過重"
print(f"你的 BMI 值為: {bmi:.2f},判定為:{x}")

 請輸入身高cm:170
 請輸入體重kg:50
你的 BMI 值為: 17.30,判定為:過輕


題目：計算BMI值，讓使用者輸入身高與體重並計算BMI及顯示分類狀況。BMI<18.5為過輕，BMI在18.5-24為正常，BMI>=24為過重，公式：BMI = 體重(kg) / 身高(m)2    # 1cm=0.01m (結果格式化輸出小數後2位）


輸入範例：
請輸入身高（公分）：170
請輸入體重（公斤）：50


輸出範例：
你的 BMI 值為：17.30，判定為：過輕

